solver/src/solver/http_solver.rs

```diff
@@ -293,10 +293,12 @@ impl HttpSolver {
         let now = chrono::Utc::now();
         format!(
             "{}_{}_{}",
-            now.to_string().replace(" ", "_"),
+            now.to_string(),
             self.network_name,
             self.chain_id
         )
+        .replace(" ", "_")
+        .replace("/", "_")
```

Comment:
Instead of manual replacements I think we should have a whitelist of characters. Something like:
1. keep only ascii
2. convert to lower case
3. keep only characters in the range 0-9 and a-z and `-`, `_`,

This is more future proof because it will not break if in the future we get weird path breaking characters.

I also think this should actually be enforced in the solver instead of in Rust because only it can know what path names are valid. Also it is a potential attack vector. What if we name the instance `../../../../../../etc/passwd`? It might allow a user of the api to take over the sovler somehow.

Reply:
Thanks for the suggestions. I agree (except with the last point - let's not attack ourselves!) , but since this is spamming the alerts channel a lot, would it be fine if we merge this as is and then do a proper fix later?

Reply:
Yes, i'm fine with it.
Of course we will not attack ourself but this is an important api design decision. What if in the future we for some reason start allowing external people to solve some instances. We want it to be secure by default.

Reply:
You're right, this is safe only as long as we're currently naming the instances ourselves.